content/en/error_tracking/backend/exception_replay.md

```diff
@@ -1,29 +1,32 @@
 ---
-title: Error Tracking Exception Replay
+title: Exception Replay in Error Tracking
 is_beta: true
 description: Learn about Exception Replay in Error Tracking.
 further_reading:
+- link: 'https://www.datadoghq.com/blog/exception-replay-datadog/'
+  tag: 'Blog'
+  text: 'Simplify production debugging with Datadog Exception Replay'
+- link: '/tracking/live-debugger'
```

Comment:
This link doesn't work for me in the preview. It goes to https://docs-staging.datadoghq.com/sarah.stonehill/exception-replay-title-fix/tracking/live-debugger, which is not a valid page.

Reply:
```suggestion
- link: '/tracing/live_debugger'
```